service/systemd/service.go

```diff
@@ -15,9 +15,12 @@ import (
 	"github.com/juju/loggo"
 	"github.com/juju/utils/shell"
 
+	"github.com/juju/juju/juju/paths"
 	"github.com/juju/juju/service/common"
 )
 
+const DataDir = "/lib/systemd/system"
```

Comment:
I keep getting this one confused with the other DataDir (/var/lib/juju), can we perhaps have LibSystemdDir or the like instead?

Reply:
Yes; I'll rename it.